### ISSUE TITLE
segmentation fault while transform from ros to apollo

### ISSUE THREAD

**Issue by sinopec** [AuroAi/apollo_ros_bridge#8]
Cause by unintilized shared ptr ln  apollo_ros_bridge\cyber_ros_bridge\core\cyber_ros_bridge_core.cpp:line 281
    std::shared_ptr<apollo::drivers::PointCloud> pc_apollo
maybe should be
    auto pc_apollo = std::make_shared<apollo::drivers::PointCloud>();

**Comment by abhilashbalachandran** [comment]
@sinopec true. Did you check if it works with your fix? I could do that, but it might take sometime. If it does, you could open a Pull Request.

**Comment by YoushaaMurhij** [comment]
Any update on this issue?
